packages/api/tests/models/rules/test-sns-rules.js

```diff
@@ -43,28 +53,21 @@ test.before(async () => {
       {}
     ),
   ]);
+});
 
-  sandbox = sinon.createSandbox();
-  sandbox.stub(awsServices, 'lambda')
-    .returns({
-      addPermission: () => ({
-        promise: () => Promise.resolve(),
-      }),
-      removePermission: () => ({
-        promise: () => Promise.resolve(),
-      }),
-    });
+test.beforeEach(async (t) => {
+  const topic = await awsServices.sns().createTopic({ Name: randomId('sns') }).promise();
+  t.context.snsTopicArn = topic.TopicArn;
```

Comment:
Can we also delete these topics in an afterEach?

Reply:
75c015420271175afb654a48ef9a6761342a4421